editor/editor_node.cpp

```diff
@@ -7079,7 +7084,6 @@ EditorNode::EditorNode() {
 	// This list is alphabetized, and plugins that depend on Node2D are in their own section below.
 	add_editor_plugin(memnew(AnimationTreeEditorPlugin));
 	add_editor_plugin(memnew(AudioBusesEditorPlugin(audio_bus_editor)));
-	add_editor_plugin(memnew(AudioStreamEditorPlugin));
```

Comment:
This class is probably no longer used.

Reply:
Good thing you noticed, seems I forgot to remove the file when extracting this part of the original PR.